setup.py

```diff
@@ -15,7 +15,7 @@ def read(fname):
     author = "Linux Foundation and SPDX Contributors",
     author_email = "opensource@steenbe.nl",
```

Comment:
Off-topic in this PR but should we change author_email from my personal email to SPDX tech mailing list?

Reply:
Yes, please change this to spdx-tech email rather than a specific person.  @swinslow can you update?

Reply:
Sure, updating now.